code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -208,3 +208,18 @@
 			return 30
 
 	return damage
+
+/mob/living/carbon/human/proc/is_noble()
+	var/noble = FALSE
+	if (job in GLOB.noble_positions)
+		noble = TRUE
+	if (HAS_TRAIT(src, TRAIT_NOBLE))
```

Comment:
i dont like that it will affect people who was given nobility during round and those who rolled that trait